src/pybind/mgr/dashboard/frontend/src/app/shared/enum/health-icon.enum.ts

```diff
@@ -1,6 +1,6 @@
 export enum HealthIcon {
   HEALTH_ERR = 'fa fa-exclamation-circle',
-  HEALTH_WARN = 'fa fa-exclamation-triangle',
+  HEALTH_WARN = 'warning--alt--filled', // Notification warning
```

Comment:
```suggestion
  HEALTH_WARN = 'warning--alt--filled', // Health warning
```

but probably the comment is not needed since its already deduced from the file that its the health icon

Reply:
@prgoel-code can this be updated. Very small nitty.